vtex-apps/slider-layout
# added centerModeSlidesGap prop

#### What problem is this solving?

When using centerMode prop on big sliders, the gap between them is too big and cannot be changed. Using the new prop, the user can set any gap he wants.

#### How to test it?

Just link the app and use the prop `centerModeSlidesGap` to change the gap between slides. Keep in mind this is only working alongside `centerMode` set to `center` or `to-the-left`

[Workspace](https://claudiuslider--elefant.myvtex.com/) - Main slider
[Workspace Floriaro](https://claudiuslides--floriaro.myvtex.com/) - On tablet, the deals under the images slider (see image below)
[Workspace Iviteb](https://claudiuslides--iviteb.myvtex.com/) - The slider below the info-card, with six images

#### Screenshots or example usage:

![before](https://user-images.githubusercontent.com/73105476/123858028-b517f780-d92b-11eb-85c9-1cc92d676b39.png)
![after](https://user-images.githubusercontent.com/73105476/123858043-b8ab7e80-d92b-11eb-8f44-fd294f58d560.png)

Screenshot floriaro:
![image](https://user-images.githubusercontent.com/73105476/131646361-66f51bf3-7f5d-4daf-8228-33684e738be2.png)

Screenshot iviteb workspace:
![image](https://user-images.githubusercontent.com/73105476/131646840-327339fd-aded-4d45-914e-3443cf1ff3ba.png)




#### Describe alternatives you've considered, if any.
